packages/coverage-istanbul/src/provider.ts

```diff
@@ -172,7 +170,7 @@ export class IstanbulCoverageProvider extends BaseCoverageProvider implements Co
 
     for (const coveragePerProject of this.coverageFiles.values()) {
       for (const filenames of [coveragePerProject.ssr, coveragePerProject.web]) {
```

Comment:
```suggestion
      for (const filenames of [coveragePerProject.ssr, coveragePerProject.client]) {
```

But as the `serverEnvironment` can be anything, I think we'll need to dynamically loop over these. 🤔